resources/protocols/intro-to-disinfection-and-sterilization.md

```diff
@@ -435,8 +445,7 @@ Sterilization indicators monitor the sterilization process and help ensure that
   <tbody>
     <tr>
       <td data-label="Type">
-        <span class="type-name">Type 1: Process Indicators</span>
-        <span class="alt-name">External indicators</span>
```

Comment:
These classes don't exist anywhere in the CSS

Reply:
And if we need specific text treatment, we can use the typography utilities:

https://getbootstrap.com/docs/5.3/content/typography/